runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangePlugin.java

```diff
@@ -679,6 +682,54 @@ public void onScriptCallbackEvent(ScriptCallbackEvent event)
 		stringStack[stringStackSize - 1] += titleBuilder.toString();
 	}
 
+	private void setLimitReset(int itemId)
+	{
+		String lastLimit = getLimitReset(itemId);
+		if (!lastLimit.equals("None"))
+		{
+			return; // do not update if previous limit has not expired
+		}
+		String itemIdStr = Integer.toString(itemId);
+		Instant currentDateTime = Instant.now();
+		configManager.setConfiguration("gelimitreset",
+			client.getUsername().toLowerCase() + itemIdStr, currentDateTime);
```

Comment:
Don't use a custom key, there is no "gelimitreset" plugin. As Adam mentioned, this should probably use the form "grandexchange", "limit.id.username". `setOffer` should have done this as well, so it's not a good example to follow.

Reply:
i misunderstood what he meant. will change that